tests/testthat/test-derive_var_confirmation_flag.R

```diff
@@ -0,0 +1,183 @@
+library(tibble)
+data <- tribble(
+  ~USUBJID, ~AVISITN, ~AVALC,
+  "1",      1,        "PR",
+  "1",      2,        "CR",
+  "1",      3,        "CR",
+  "1",      4,        "SD",
+  "1",      5,        "NE",
+  "2",      1,        "SD",
+  "2",      2,        "PR",
+  "2",      3,        "PD",
+  "3",      1,        "SD",
+  "4",      1,        "PR",
+  "4",      2,        "PD",
+  "4",      3,        "SD",
+  "4",      4,        "SD",
+  "4",      5,        "PR"
+)
+
+# derive_var_confirmation_flag ----
+## Test 1: filter without first_cond ----
```

Comment:
might help to add an extra line of more detail of the derivation you are intending in these test comment explanations, e.g. Flagging any patient PR value that is followed subsequently by a CR or PR

Reply:
do same for newly added Test 5 & 6

Reply:
Done